crates/uv-resolver/src/lock.rs

```diff
@@ -1464,26 +1464,29 @@ impl TryFrom<WheelWire> for Wheel {
 
 /// A single dependency of a distribution in a lock file.
 #[derive(Clone, Debug, Eq, PartialEq, PartialOrd, Ord, serde::Deserialize)]
-struct Dependency {
+struct DependencyId {
```

Comment:
When I see `DependencyId`, I think, "what is it an ID for? what is it identifying?" For a `DistributionId`, it's used to lookup a `Distribution`. What can a `DependencyId` lookup?

Reply:
Ok. I can change it back to `Dependency`, but I don't think the field on it should be called `id`, since that's not an ID for the _dependency_.

Reply:
Yeah that's fair.